dev/yarn.ts

```diff
@@ -121,6 +121,30 @@ const configList: Fig.Generator = {
 export const completionSpec: Fig.Spec = {
   name: "yarn",
   description: "Manage packages and run scripts",
+  generateSpec: async (context, executeShellCommand) => {
```

Comment:
```suggestion
  generateSpec: async (_context, executeShellCommand) => {
```

TypeScript suggests prefixing unused params with `_`